### PR TITLE
Add bzip2

### PR DESCRIPTION
**Pull request by jsharpe**

### CODE REVIEW

File: modules/bzip2/1.0.8/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
++        "huffman.c",
++        "randtable.c",
++    ],
++    hdrs = [
++        "bzlib.h",
++    ],
++    copts = [
++        "-D_FILE_OFFSET_BITS=64",
++    ],
++    strip_include_prefix = "",
```
Comment (fmeum) [inline modules/bzip2/1.0.8/patches/add_build_file.patch:34]: Does an empty `strip_include_prefix` have an effect here?

**Review by fmeum — APPROVED** [review]

File: modules/bzip2/1.0.8/patches/add_build_file.patch (hunk 1 above)
Comment (jsharpe) [reply modules/bzip2/1.0.8/patches/add_build_file.patch:34]: Good question - this came from the dropbox dbx_build_tools repo originally.

Comment (jsharpe) [reply modules/bzip2/1.0.8/patches/add_build_file.patch:34]: Turns out it does - its working around the fact that bzip2 internally uses <> includes

**Comment by jsharpe** [comment]
Seems like CI was broken when I rebased this change; I don't have permissions to retrigger the build.

**Comment by fmeum** [comment]
Me neither, but you should be able to amend without changes and force-push.

**Review by fmeum — APPROVED** [review]

**Review by fmeum — APPROVED** [review]

**Pull request merged**